packages/shared/src/variants/deprecated/badukWithAbstractBoard.ts

```diff
@@ -52,18 +52,12 @@ export function mapToNewBadukConfig(config: unknown): NewBadukConfig {
   return { komi: typedConfig.komi, board: mapToNewBoardConfig(typedConfig) };
 }
 
-export const badukWithAbstractBoardVariant: Variant<BadukConfig> = {
-  gameClass: Baduk,
+export const badukWithAbstractBoardVariant: Variant<
+  NewBadukConfig,
+  BadukState
+> = {
+  ...badukVariant,
```

Comment:
This variant is functionally equivalent to Baduk, so I think copying in everything (except sanitizeConfig) makes sense.  I didn't think of a good way to test though :grimacing: